egs2/fsc/asr1/README.md

```diff
@@ -25,8 +25,18 @@
 - keep_nbest_models: 5
 - Pretrainded model: https://zenodo.org/record/5171544
 
-|dataset|Snt|S.Err|
+|dataset|Snt|Intent Classification (%)|
 |---|---|---|
 |inference_asr_model_valid.acc.ave_5best/test|3793|98.8|
 |inference_asr_model_valid.acc.ave_5best/valid|3118|93.6|
 
+
+## Using Transformer based encoder-decoder and decoding sentence with spectral augmentation and predicting transcript along with intent
```

Comment:
Can you also add an ASR result (just as a record)?

Reply:
I have added the ASR results but they are over the entire text i.e. over the intent concatenated with transcript. Should I add another scoring script to compute these metrics only over transcript (i.e. after excluding the first word)?

Reply:
I think having ASR results without the intent is better.